tests/ryzenai/operators_baseline.json

```diff
@@ -0,0 +1,1657 @@
+{
```

Comment:
You could add a reference somewhere as to which RyzenAI version this corresponds to, if you want.

Reply:
Added in `testing_utils.py` and `README.md` with link to steps to generate one.